packages/web-components/package.json

```diff
@@ -0,0 +1,100 @@
+{
+  "name": "@microsoft/fast-components-msft",
+  "description": "A library of Fluent Web Components",
+  "sideEffects": false,
+  "version": "1.6.0",
+  "author": {
+    "name": "Microsoft",
+    "url": "https://discord.gg/FcSNfg4"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/fast.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Microsoft/fast/issues/new/choose"
+  },
+  "main": "dist/esm/index.js",
+  "types": "dist/fast-components-msft.d.ts",
+  "unpkg": "dist/fast-components-msft.min.js",
+  "scripts": {
+    "clean:dist": "node ./build/clean.js dist",
+    "doc": "api-extractor run --local",
+    "doc:ci": "api-extractor run --local",
+    "build": "tsc -p ./tsconfig.json && rollup -c && npm run doc",
+    "dev": "tsc -p ./tsconfig.json -w",
+    "tdd": "npm run dev & npm run test-chrome:watch",
+    "prepare": "yarn clean:dist && yarn build",
+    "prettier": "prettier --config ../../prettier.config.js --write \"**/*.{ts,html}\"",
```

Comment:
Looks like there are some differences between the prettier config of the incoming files and the one for the whole repo (2 vs 4 space most obviously). If you're okay with reformatting it would be nice to have consistency, or if not can you add an override to the root `prettier.config.js`?

Reply:
I ran prettier and this references the Fluent project config. It could be a different version of prettier that's causing the issues. I'll take a look.